docs/user_guide/mitigation/reductions.rst

```diff
@@ -551,12 +551,88 @@ Group :code:`"a"` has an average loss of :math:`0.05`, while group
 
 .. note::
 
-    In the example above the :code:`BoundedGroupLoss` object does not use the
+    In the example above the :code::class:BoundedGroupLoss` object does not use the
```

Comment:
The syntax looks off. Or does this work?

Reply:
oh my, that is some weird editor hiccup that I didn't catch somehow by checking the diff. very likely a misplaced paste :S. it should just stay as it was for now :) thanks for catching it!